lib/matplotlib/image.py

```diff
@@ -144,7 +144,11 @@ def flush_images():
                     gc = renderer.new_gc()
                     gc.set_clip_rectangle(parent.bbox)
                     gc.set_clip_path(parent.get_clip_path())
-                    renderer.draw_image(gc, round(l), round(b), data)
+                    if type(renderer) == mpl.backends.backend_agg.RendererAgg:
+                        renderer.draw_image(gc, l, b, im)
+                    else:
+                        renderer.draw_image(gc, round(l), round(b), data,
+                                            bbox=parent.bbox)
```

Comment:
Turns out this is wrong (the test failure test_image.py:test_bbox_image_inverted is real), since BboxImages are allowed to fall outside the axes (parent) bbox. 

There are two ways that `draw_image` gets called. By `stop_rasterizing` and by `(_ImageBase).draw` (here). The latter is correctly dealt with in this PR already, but the former needs a more "correct" fix than passing the parent bbox.